step.yml

```diff
@@ -1,6 +1,29 @@
-title: Android Lint
-summary: This step runs lint gradle task on your project.
-description: This step runs lint gradle task on your project.
+title: "Android Lint"
+summary: |-
+  Runs Lint on your Android build's code and detects potential syntax errors to keep your code error free.
+description: |-
+  It highlights the code line where the error is found, explains the type of error and suggests correction. This Step does not make the build fail if it spots any structural errors in your code. The results are available in a viewable or downloadable Lint Report html file which you can access on the Build's APPS & ARTIFACTS page.
```

Comment:
Lint Report html file -> Lint Report HTML or XML file

(see: report_path_pattern/summary)

Reply:
which you can access on the Build's APPS & ARTIFACTS page. -> should not we mention that this needs to have the Deploy to Bitrise.io step in the workflow? (without that step it is not true)

Reply:
will do :)